x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/index.tsx

```diff
@@ -59,51 +60,56 @@ export const HostsPage = () => {
 
   return (
     <EuiErrorBoundary>
-      <div className={APP_WRAPPER_CLASS}>
-        <MetricsPageTemplate
-          hasData={metricIndicesExist}
-          pageHeader={{
-            alignItems: 'center',
-            pageTitle: (
-              <div
-                css={css`
-                  display: flex;
-                  align-items: center;
-                  gap: 0.75rem;
-                `}
-              >
-                <h1>{hostsTitle}</h1>
-                <BetaBadge
-                  tooltipContent={i18n.translate('xpack.infra.hostsViewPage.betaBadgeDescription', {
-                    defaultMessage:
-                      'This feature is currently in beta. If you encounter any bugs or have feedback, we’d love to hear from you. Please open a support issue and/or share your feedback via the "Tell us what you think!" feedback button.',
-                  })}
-                />
-              </div>
-            ),
-            rightSideItems: [
-              <FeatureFeedbackButton
-                data-test-subj="infraHostsPageTellUsWhatYouThinkButton"
-                formUrl={HOSTS_FEEDBACK_LINK}
-                kibanaVersion={kibanaVersion}
-                isCloudEnv={isCloudEnv}
-                isServerlessEnv={isServerlessEnv}
-              />,
-            ],
-          }}
-          pageSectionProps={{
-            contentProps: {
-              css: fullHeightContentStyles,
-            },
-          }}
-        >
-          {source && (
-            <MetricsDataViewProvider metricAlias={source.configuration.metricAlias}>
-              <HostContainer />
-            </MetricsDataViewProvider>
-          )}
-        </MetricsPageTemplate>
-      </div>
+      <PerformanceContextProvider>
```

Comment:
Does it make sense to have this context defined on a broader scope? like here https://github.com/elastic/kibana/blob/5b38758ff27df55c3fc1839539b4a4612608b2d7/x-pack/plugins/observability_solution/infra/public/apps/common_providers.tsx#L26?

Reply:
Yes 100%, let me update it

Reply:
updated https://github.com/elastic/kibana/pull/180309/commits/876fa08a30fe977ad9a54ff396d68f3bd653c8b0